packages/charts/react-charts/library/src/components/CommonComponents/CartesianChart.tsx

```diff
@@ -309,26 +285,25 @@ export const CartesianChart: React.FunctionComponent<ModifiedCartesianChartProps
     _xScale = xScale;
     _tickValues = tickValues;
 
-    /*
-     * To enable wrapping of x axis tick values or to display complete x axis tick values,
-     * we need to calculate how much space it needed to render the text.
-     * No need to re-calculate every time the chart renders and same time need to get an update. So using set
-     * Required space will be calculated first time chart rendering and if any width/height of chart updated.
-     * */
-    if (props.wrapXAxisLables || props.showXAxisLablesTooltip) {
-      const wrapLabelProps = {
-        node: xAxisElement.current!,
-        xAxis: xScale,
-        showXAxisLablesTooltip: props.showXAxisLablesTooltip || false,
-        noOfCharsToTruncate: props.noOfCharsToTruncate || 4,
-      };
-      const temp = xScale && (createWrapOfXLabels(wrapLabelProps) as number);
-      // this value need to be updated for draw graph updated. So instead of using private value, using set
-      if (isRemoveValCalculated && removalValueForTextTuncate !== temp) {
-        setRemovalValueForTextTuncate(temp);
-        setIsRemoveValCalculated(false);
-      }
-    }
+    _transformXAxisLabels();
+
+    const YAxisParams = {
```

Comment:
this PR doesn't seem to be in chronological order. The original PR is of June 20th